### PR TITLE
Customer Stories

### LINKED ISSUE #645: New customer story
### Story

_No response_

### Description

We want to start publishing stories about companies using Node-RED. This is the first draft of a manufacturing company in the US. The company can not be named.

https://docs.google.com/document/d/1BDSkrhFNe6aB4bCfs13vPzZ1euSO2OY4wz3kFWjceLE/edit?usp=sharing

### PR DESCRIPTION
**Pull request by joepavitt**
## Description

- Add new `/customer-stories` section to website
- Add first customer story (with new template)

## Related Issue(s)

Closes #645 

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/flowforge/flowforge/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes

### CODE REVIEW

File: src/customer-stories.njk [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        {%- for item in collections.stories | reverse -%}
+        {% if loop.first %}
+        <li class="w-full md:w-2/3 mt-2 px-2 pb-4">
+            <a href="{{ item.url }}" class="w-full flex flex-col group hover:no-underline">
+                <div class="pr-2">
+                    {% if item.data.image %}
+                        <div class="w-full h-60 md:h-72">
+                            {% set imageSrc = [item.data.image ] | join %}
+                            {% set imageDescription = ["Image representing ", item.data.title] | join %}
+                            <img src="{{ imageSrc }}" alt="{{ imageDescription }}" class="object-cover w-full h-full" />
```
Comment (joepavitt) [inline src/customer-stories.njk:22]: @Pezmc thoughts on this please? The shortcode doesn't allow us to pass CSS classes to it, here we want to make the most of Tailwind's `object-cover`. We could switch the layout to be using a background-image instead, but feles excessive.

Comment (Pezmc) [reply src/customer-stories.njk:22]: @joepavitt Can you apply the CSS directly? As it stands this image won't go through the image pipeline and get resized at all.

``css
.object-cover {
    object-fit: cover;
}
```

Comment (joepavitt) [reply src/customer-stories.njk:22]: yes, but on what? the `image` shortcode doesn't let us apply styles either, right?

Comment (Pezmc) [reply src/customer-stories.njk:22]:
```
 <div class="w-full h-60 md:h-72 image-cover">
   {{ image-tag }}
 </div>
 ```
 
 ```css
.image-cover > img {
    object-fit: cover;
}
```

Comment (joepavitt) [reply src/customer-stories.njk:22]: oh that's dirty - it'll work though

**Comment by joepavitt** [comment]
@Pezmc I'll action that `object-cover` later, focussing on Product right now, but if you could put some eyes on the rest whenever you get a chance, that'd be grand.

**Review by Pezmc — APPROVED** [review]

**Pull request merged**